lib/streamlit/script_runner.py

```diff
@@ -166,18 +171,9 @@ def start(self) -> None:
             raise Exception("ScriptRunner was already started")
 
         self._script_thread = threading.Thread(
-            target=self._process_request_queue,
+            target=self._run_script_thread,
             name="ScriptRunner.scriptThread",
         )
-
-        script_run_ctx = ScriptRunContext(
-            session_id=self._session_id,
-            enqueue=self._enqueue_forward_msg,
-            query_string=self._client_state.query_string,
-            session_state=self._session_state,
-            uploaded_file_mgr=self._uploaded_file_mgr,
-        )
-        add_script_run_ctx(self._script_thread, script_run_ctx)
         self._script_thread.start()
```

Comment:
ScriptRunContext is now created inside the script thread itself, which feels a bit cleaner - it doesn't need to be created until the thread starts executing.